src/Application/UseCases/CreatePersistentPageIdentifier.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace ProfessionalWiki\PersistentPageIdentifiers\Application\UseCases;
+
+use ProfessionalWiki\PersistentPageIdentifiers\Application\PersistentPageIdentifiersRepo;
+use ProfessionalWiki\PersistentPageIdentifiers\Infrastructure\IdGenerator;
+
+class CreatePersistentPageIdentifier {
+
+	public function __construct(
+		private readonly PersistentPageIdentifiersRepo $repo,
+		private readonly IdGenerator $idGenerator
+	) {
+	}
+
+	public function createId( int $pageId ): void {
+		$this->repo->savePersistentId( $pageId, $this->idGenerator->generate() );
+	}
+
+}
```

Comment:
I'm not sure if we'll need to handle errors. Otherwise, I don't expect anything else to go in here, but I do expect the use case to be used in the maintenance script.